management/src/main/java/org/kaazing/gateway/management/filter/FullManagementFilterStrategy.java

```diff
@@ -61,7 +61,6 @@ public void doSessionClosed(ManagementContext managementContext,
         if (sessionBean != null) {
             managementContext.removeSessionManagementBean(sessionBean);
             managementContext.getManagementSessionStrategy().doSessionClosed(sessionBean);
-            managementContext.getManagementSessionStrategy().doSessionClosed(sessionBean);
         }
```

Comment:
No need to do this twice.  I think it was added due to notifications not being delivered properly.  This doesn't fix that and has the resulting NPE appear twice.